static/js/public/snap-details/channelMap.js

```diff
@@ -146,13 +148,26 @@ function initOpenSnapButtons() {
       document.body.appendChild(iframe);
 
       if (typeof ga !== 'undefined') {
+        // The first attempt should be counted towards the 'intent'
+        let label = 'Snap install intent';
+        let value = `${name}`;
+
+        // Subsequent attempts should still be tracked, but not as 'intent'
+        if (attempt > 1) {
+          label = 'Snap install click';
+          value += ` - click ${attempt}`;
```

Comment:
We don't want snap `name` in next attempts label?

Reply:
`value +=`

Reply:
Ah, it's `+=`, so never mind...